src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10785,6 +10802,166 @@ namespace OutputReportTabular {
 		}
 	}
 
+    // Parses the contents of the EIO (initializations) file and creates subtables for each type of record in the tabular output files
+	// Glazer - November 2016
+	void
+	WriteEioTables() {
+
+		if (displayEioSummary)
+		{
+			// all arrays are in the format: (row, column)
+			Array1D_string columnHead;
+			Array1D_int columnWidth;
+			Array1D_string rowHead;
+			Array2D_string tableBody;
+			Array1D_int colUnitConv;
+
+			// setting up  report header 
+			WriteReportHeaders("Initialization Summary", "Entire Facility", isAverage);
+
+			// since the EIO initilization file is open at this point must close it to read it and then reopen afterward.
+			gio::close(OutputFileInits); 		// for testing only
+
+			std::ifstream eioFile;
```

Comment:
Hmm.  I didn't quite realize this was the approach.  Reading output files that E+ just wrote.  In an ideal world, it would've been great to create a very lightweight class, and when the eio is written by a component, it first populates an instance of the class, then writes the eio.  Then anyone who wants to do anything with the eio data could just grab from that same in-memory data, instead of having to re-read the eio.  Even with two simplifications (instantiate the class globally so you can access it from anywhere, and have it pretty much just store an array of strings, one per line), it would be very straightforward to implement and just much better than reading back the eio data.  What would you think about making this ticket a higher complexity, but pursuing something like this?